doc/02-Installation.md

```diff
@@ -202,6 +203,80 @@ After creating the database, you can import the Icinga DB schema using the follo
 mysql -u root -p icingadb </usr/share/icingadb/schema/mysql/schema.sql
 ```
 
+### Configuring PostgreSQL <a id="configuring-icingadb-pgsql"></a>
+
+#### Installing PostgreSQL database server <a id="installing-database-pgsql-server"></a>
+
+RHEL/CentOS 7:
+
+```
+yum install centos-release-scl
+yum install rh-postgresql95-postgresql-{server,contrib} postgresql
+/opt/rh/rh-postgresql95/root/usr/bin/postgresql-setup initdb
+
+systemctl enable rh-postgresql95-postgresql
+systemctl start rh-postgresql95-postgresql
+```
+
+RHEL/CentOS 8:
+
+```
+yum install postgresql-{server,contrib} postgresql
+postgresql-setup initdb
+
+systemctl enable postgresql
+systemctl start postgresql
+```
+
+SUSE:
+
+```
+zypper install postgresql-{server,contrib} postgresql
+
+systemctl enable postgresql
+systemctl start postgresql
+```
+
+Debian/Ubuntu:
+
+```
+apt install postgresql postgresql-client
+```
+
+#### Setting up the PostgreSQL database <a id="setting-up-pgsql-db"></a>
```

Comment:
I would only speak of `pg_hba.conf` and not list all alternative paths for the different operating systems.